ugvc/mrd/featuremap_consensus_utils.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from os.path import basename
+from os.path import join as pjoin
+
+import numpy as np
+import pysam
+
+
+def write_a_pileup_record(
+    record_dict: dict, rec_id: str, out_fh: pysam.VariantFile, header: pysam.VariantHeader, min_qual: int
+):
+    info_fields = [
+        "X_QUAL",
+        "X_RN",
+    ]
+
+    format_fields = ["GT", "AD", "DP", "VAF"]
+
+    rec = header.new_record()
+    rec.chrom = rec_id.split("-")[0]
+    rec.pos = int(rec_id.split("-")[1])
+    rec.ref = rec_id.split("-")[2]
+    rec.alts = [rec_id.split("-")[3]]
+    rec.id = "."
+    rec.qual = np.mean(record_dict["X_QUAL"])
```

Comment:
I don't think the mean is the best aggregation, since additional supporting reads can decrease it. I think even the highest qual would be better, until we have a more refined algorithm.

Maybe it's also good to have some support for a quality aggregation function as an input.